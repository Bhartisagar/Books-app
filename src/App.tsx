import React from 'react';
import Template from './components/templates';
import { BrowserRouter as Router, Route,Routes } from 'react-router-dom';
import Entrepreneurship from './components/templates/Entrepreneurship';
import Bookdetailviewpage from './components/templates/Bookdetailviewpage';
import CoverPage from './pages/CoverPage';
import Theme from './Theme/Theme'
import { ThemeProvider } from '@material-ui/core';

function App() {
  return (  
    //  <Template/>
    <ThemeProvider theme={Theme}>
    <Router>
       <Routes>
        <Route path="/" element={<CoverPage/>} />
        
        <Route path='/library' element={<Template/>}/>

        <Route path='/entrepreneurship' element={<Entrepreneurship/>}> </Route>

        <Route path='/bookdetailviewpage/:id' element={<Bookdetailviewpage/>}> </Route>
      
        </Routes>

    </Router>
    </ThemeProvider>
     
   
  )
}

export default App;
