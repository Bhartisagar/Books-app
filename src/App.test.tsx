import App from "./App";

import renderer from 'react-test-renderer'

test("first snapshot",()=>{
    const component = renderer.create(
        
        <App></App>
       
    )
    let tree=component.toJSON();

    expect(tree).toMatchSnapshot();
})