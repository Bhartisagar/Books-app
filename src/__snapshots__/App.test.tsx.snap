// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first snapshot 1`] = `
<div
  className="makeStyles-main-1"
>
  <div
    className="makeStyles-logo-2"
  >
    <img
      alt="blinkistcoverlogo"
      height={770}
      src="image 1.png"
      width={950}
    />
    <h2
      className="MuiTypography-root makeStyles-heading-3 MuiTypography-h2"
    >
      LDP Dev interns-
    </h2>
    <h2
      className="MuiTypography-root makeStyles-heading2-4 MuiTypography-h2"
    >
      Blinkist ps
    </h2>
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-buttons-5 MuiButton-outlinedSizeLarge MuiButton-sizeLarge"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Login
    </span>
  </button>
</div>
`;
